### ISSUE TITLE
Fix Scilab unit test failures

### ISSUE THREAD

**Issue by khumba** [khumba/cgal-swig-bindings#7]
~80% of the Scilab unit tests passing currently (as of f4447946).  Most of the failures are Arrangement_2 tests.  CC @Deepaknkumar.

**Comment by Deepaknkumar** [comment]
@khumba As of Commit f444794, All the tests are failing on Ubuntu 14.04 platform, Also

 exec('build/SWIG_CGAL/Kernel/sciloader.sce', -1);
echos,

exec('build/SWIG_CGAL/Kernel/sciloader.sce', -1);
Shared archive loaded.
Link done.

not,

Warning :
exec('build/SWIG_CGAL/Kernel/sciloader.sce', -1);
The identifier : libCGAL_Kernel_Scilab_path
 has been truncated to: libCGAL_Kernel_Scilab_pa.

Warning :

The identifier : libCGAL_Kernel_Scilab_path
 has been truncated to: libCGAL_Kernel_Scilab_pa.

Shared archive loaded.
Link done.

as in Gentoo case. Hence we need to check what exec commands prints on terminal, which is different on other platforms!

**Comment by khumba** [comment]
Ah yeah, that difference would make them all fail :(.  Is that output from Scilab 6 where the name limit doesn't exist anymore (does it work if you run on Scilab 5)?  I recall looking into the SWIG code that generates that variable name a while back, and I didn't see an easy way to shorten it.  Would be nice to fix that warning though.

**Comment by khumba** [comment]
Hmm, if this differs between 5.x versions then we might need to use
`// <-- NO CHECK REF -->` and do assertions manually.

**Comment by khumba** [comment]
Still failing tests:

```
004/120 - [/kh/files/dev/projects/cgal-swig-bindings.git] Arrangement_2_insert_in_face_interior failed: premature end of the test script
007/120 - [/kh/files/dev/projects/cgal-swig-bindings.git] Arrangement_2_merge_edge failed: the ref file doesn't exist
014/120 - [/kh/files/dev/projects/cgal-swig-bindings.git] Arrangement_2_number_of_isolated_verts failed: premature end of the test script
016/120 - [/kh/files/dev/projects/cgal-swig-bindings.git] Arrangement_2_number_of_vertices failed: premature end of the test script
017/120 - [/kh/files/dev/projects/cgal-swig-bindings.git] Arrangement_2_number_of_verts_at_infty failed: premature end of the test script
020/120 - [/kh/files/dev/projects/cgal-swig-bindings.git] Curve_2_line failed: the ref file doesn't exist
021/120 - [/kh/files/dev/projects/cgal-swig-bindings.git] Curve_2_new failed: premature end of the test script
047/120 - [/kh/files/dev/projects/cgal-swig-bindings.git] Line_2_is_horizontal failed: dia and ref are not equal
048/120 - [/kh/files/dev/projects/cgal-swig-bindings.git] Line_2_is_vertical failed: dia and ref are not equal
```

The Line_2 failures at least are because of bug #4.

**Comment by khumba** [comment]
I've fixed the remaining broken unit tests (013ef9e28).  These require the patch from swig/swig#534 to work though.

**Comment by khumba** [comment]
Closing, as there's nothing else to be done on this issue directly.
